default.yaml

```diff
@@ -133,6 +133,10 @@ ipRecording:
 
 # blacklistedIPs: ['1.2.3.4', '5.6.7.8']
```

Comment:
not fully related to this PR, but I think this comment can be removed (and maybe the one above as well?)

we initially had in mind to do some filtering at the app the level, but this should really be done at the nginx/load balancer level